src/main/java/com/umc/networkingService/global/common/exception/ErrorCode.java

```diff
@@ -28,8 +28,12 @@ public enum ErrorCode {
     UNAUTHORIZED_UPDATE_CENTER_POSITION(HttpStatus.BAD_REQUEST, "MEMBER003", "지부, 학교 운영진은 중앙 직책을 부여할 수 없습니다."),
     EMPTY_MEMBER_UNIVERSITY(HttpStatus.CONFLICT, "MEMBER004", "소속 대학교가 존재하지 않는 사용자입니다."),
     UNAUTHENTICATED_GITHUB(HttpStatus.BAD_REQUEST, "MEMBER005", "깃허브 연동이 완료되지 않은 사용자입니다."),
+
+    NO_PERMISSION_MEMBER(HttpStatus.BAD_REQUEST, "MEMBER006", "권한이 없는 사용자입니다."),
```

Comment:
아래에 이미 만들어진 Errorcode와 코드가 중복이어서, 코드를 변경해주는 게 좋을 거 같아요!! 그리고 TIL에 대해서 따로 만들어 주는 건 어떨까요?
```suggestion
    UNAUTHORIZED_TODAY_I_LEARNED(HttpStatus.BAD_REQUEST, "TIL001", "해당 TIL에 대해 권한이 없는 사용자입니다."),
```

Reply:
수정 완료